WeChat/CustomViewCOntroller.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CustomViewCOntroller.swift
+//  WeChat
+//
+//  Created by Devesh on 04/05/24.
+//
+
+import UIKit
+
+class CustomViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .green
```

Comment:
We do not need to write `self.view` here. It should work with just view.background......